src/tests/v110/test_list_collections.py

```diff
@@ -0,0 +1,23 @@
+import fireo
+from fireo.fields import IDField
+from fireo.models import Model
+
+
+class Doc(Model):
+    doc_id = IDField(db_name='id')
+
+def _create_collections(count: int):
+    collections = []
+    for c in range(count):
+        collection = f"collection_{c}"
+        collections.append(collection)
+        doc = Doc(doc_id=f"{c}_1", collection_name=collection)
```

Comment:
FireO is not created by automatically generate the **collections** but if you want to do that then you need to know that `collection_name` that is used to **Firestore collection** is a class variable not instance variable.  
You can do like this to automatically generate collections.  

```python
doc = Doc(doc_id=f"{c}_1")
Doc.collection_name = collection
doc.save()
````

Reply:
@AxeemHaider what you are suggesting didn't work in my testing.  I was planning on creating a new branch to figure out why, and create fix.